DANSEWELL - Reno

```diff
@@ -0,0 +1,11 @@
+Reno NV
+
+Resturaunts  (Spell-Check is not automatic)
+
+BasknRobins - IceCream is a meal
+ColdStone Creamery - 
+
+## Frozen Yogurt != Ice Cream
```

Comment:
There are a few in Sparks, one has a frog in the name.  the other is a beach theme